openedx/core/djangoapps/user_authn/views/login.py

```diff
@@ -365,7 +365,11 @@ def login_user(request):
             except AuthFailedError as e:
                 set_custom_metric('login_user_tpa_success', False)
                 set_custom_metric('login_user_tpa_failure_msg', e.value)
-                return HttpResponse(e.value, content_type="text/plain", status=403)
+
+                # user successfully authenticated with a third party provider, but has no linked Open edX account
```

Comment:
**Reviewer Note:** I think the status used (403) is debatable for this use case.  Given I don't feel strongly it should change, I am voting to leave it as-is to reduce the number of breaking changes.